test/test_group.cpp

```diff
@@ -1620,7 +1620,7 @@ TEST(Group_ChangeEmbeddedness)
     // obj2 has no owner, so we can't make the table embedded
     std::string message;
     CHECK_THROW_ANY_GET_MESSAGE(t->set_embedded(true), message);
-    CHECK_EQUAL(message, "At least one object does not have a backlink (data would get lost).");
+    CHECK_EQUAL(message, "At least one object in table 'table' does not have a backlink (data would get lost).");
```

Comment:
Now that texts have changed, tests should be adopted I imagine.

Reply:
Good that you are awake.